### ISSUE TITLE
Question about the SRN training parameters

### ISSUE THREAD

**Issue by yxgnahz** [PaddlePaddle/PaddleOCR#2127]
Hi, I notice that in your SRN paper you show the performance ONLY with PVAM module. To reproduce the result, could you provide us the training settings, e.g., init learning rate, optimizer hyper-parameters, and training epochs?
